src/hooks/series/useNextEpisode.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from 'react-query';
+
+import type { EpisodeMetadata, Series } from '#types/series';
+import { getNextItem } from '#src/utils/series';
+import { SERIES_CACHE_TIME } from '#src/config';
+
+export const useNextEpisode = ({ series, episodeMetadata }: { series: Series | undefined; episodeMetadata: EpisodeMetadata | undefined }) => {
+  const { isLoading, data } = useQuery(
+    ['next-episode', series?.series_id],
```

Comment:
Don't you need to include current episode ID in this key? Otherwise I think there will just be 1 next episode result per series.